site/docs/patterns/list-filtering.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: List filtering
+layout: DetailPattern
+aliases:
+  - /salt/patterns/list-filtering
+data:
+  components: ["Input", "List", "Spinner"]
+  relatedpatterns: ["List builder"]
+---
+
+When searching through a list, a filter enables users to specify criteria and only show relevant options. This pattern introduces the filter behavior of Salt components where a list is used.
+
+  <Diagram
+    src="/img/patterns/list-filtering/list-filter.png"
+    alt="A list filter."
+    contentPosition="top"
+    border
+  />
+
+## When to use
+
+Use filtering when you have a lengthy list that users have to scroll to view all available options. The pattern typically comprises input and list based components to use a string to filter items for quick selection.
+
+List filtering is a common behavior found in:
+* [`ComboBox`](/salt/components/combo-box)
+* [List builder](/salt/patterns/list-builder)
+* Pivot table
+* Tree builder
+
+## Filter logic 
+
+When filtering results, we recommend:
+
+* Case-insensitive matching, i.e., Alabama, alabama, ALAbama, will all return the same result.
+* Matching only the first character(s) of the beginning of any word in the item labels to the filter string.
+
+  <Diagram
+    src="/img/patterns/list-filtering/list-filter-logic.png"
+    alt="An example of filter logic."
+    contentPosition="top"
+    border
+  />
+
+<ExampleContainer type="positive" customPillText="Best practices">
+Increase the number of characters required to filter the list if there are a large number of items and results are slow to load. You could, for example, fetch results on input of the first character but only display them on input of the third. This would improve perceived performance.
```

Comment:
```suggestion
Increase the minimum number of characters required to filter the list if there are a large number of items and results are slow to load. You could, for example, fetch results on input of the first character but only display them on input of the third. This would improve perceived performance.
```